src/posts/entities/post.repository.ts

```diff
@@ -15,4 +16,12 @@ export class PostRepository extends Repository<Post> {
     post.ready = false;
     return await this.save(post);
   }
+  public async deletePost(postid: string): Promise<Post> {
+    try {
+      const post = await this.findOneOrFail({ where: { uuid: postid } });
+      return await Post.remove(post);
```

Comment:
- Please remove `return` keyword as we don't use the returned value in posts.service.ts or anywhere for now.